Gemfile

```diff
@@ -4,7 +4,7 @@ ruby '~> 2.5'
 gem 'require_all'
 
 gem 'puma'
-gem 'rack'
+gem 'rack', github: 'senhalil/rack', branch: 'improved-asserts'
```

Comment:
can you put in comment the link to related PR in order to remove this fork as soon as it's merged?